src/components/DarkModeToggle.tsx

```diff
@@ -20,7 +20,7 @@ export const DarkModeToggle = (props: {
 
   return (
     <label
-      className={`ToolIcon ToolIcon_type_floating ToolIcon_size_M`}
+      className="ToolIcon ToolIcon_type_floating ToolIcon_size_M toggle-dark-mode"
```

Comment:
```suggestion
      className="ToolIcon ToolIcon_type_floating ToolIcon_size_M"
      data-testid="toggle-dark-mode"
```
Since this class is only added for testing purpose so we can use `data-testid` instead to make it clear thats its only for test

Reply:
fixed, I used queryByTestID since that supports the case where you expect and don't expect it to appear. I didn't see other examples in the codebase but lmk if you'd like me to query it differently.

Reply:
Nice. We should use `queryByTestId` going forward 👍